### ISSUE TITLE
VerifiedKeyWithExpiration.verifier is nil

### ISSUE THREAD

**Issue by colorfulfool** [rails/activestorage#54]
Installed ActiveStorage, configured according to README, successfully uploaded a photo via `MirrorService` (local + amazon). Now,
``` 
2.3.4 :003 > ActiveStorage::VerifiedKeyWithExpiration.verifier
=> nil
```

even though
```
2.3.4 :004 > defined?(Rails) ? Rails.application.message_verifier("ActiveStorage") : nil
 => #<ActiveSupport::MessageVerifier:0x007f9ea69a3818 @secret="\x8C\eA>\x96\xC2\x91c\xBAP\xAB$%\xB5|\xBA\x9E\xD9\x8E\xB8=o\xB9\x8E\x88aI\x85g\"\xCE\x7FL+>l|\xF8S#?\xD2\xE8\xBE[g\xD8\xF7\"\x0FR\xEDBe\xE5\xE6\xB3\x9EF~\xB2\xEBJ\x14", @digest="SHA1", @serializer=Marshal>
```
As a result,
```
2.3.4 :005 > jacket.photos.first.url
  Disk Storage (9.5ms) Generated URL for file at key: TzDZEQAJdY1HBUg5Hym16oPd ()
NoMethodError: undefined method `generate' for nil:NilClass
        from (irb):5
```
Any help is welcome 🤔

**Comment by georgeclaghorn** [comment]
What version of Rails are you using?

**Comment by colorfulfool** [comment]
`rails 5.1.2`.

Setting `ActiveStorage::VerifiedKeyWithExpiration.verifier` explicitly at boot solves the problem.

**Comment by georgeclaghorn** [comment]
The `:default` option for `class_attribute` was added to Rails master in May and hasn’t been included in a released version of Rails yet.

Active Storage will be included in Rails 5.2 and, generally speaking, isn’t compatible with Rails 5.1. (It was at one point, which is why the gemspec still permits 5.1.)
